### PR TITLE
app-admin/ryzen_monitor: add myself as maintainer, maintenance

### PR DESCRIPTION
**Pull request by negril**
Updates `app-admin/ryzen_monitor` to work with latest `app-admin/ryzen_smu`.

Not sure if that is really the right way to do this, but it's the easiest. Otherwise we need to create and install a library from `ryzen_smu`.

### CODE REVIEW

**Comment by dlan17** [comment]
<+dlan> personally I'd rather not download ryzen_smu source, but just create a patch
 <+dlan> but anyway, you should really work with upstream to have a better solution(instead of carrying the patch downstream)

**Comment by negril** [comment]
`app-admin/ryzen_smu` is a `RDEPEND` of `app-admin/ryzen_monitor` and contains the corresponding kernel module, and is thus likely rebuild more often, meaning the `ryzen_smu` source is always downloaded anyway.

`ryzen_monitor` already contains a copy of the `ryzen_smu` source code. That is just replaced with the newer code from the current, now incompatible `ryzen_smu-0.1.5`.

My changes to the ebuild allow to easily adjust that. And I am trying to be the maintainer of both so I'll do that when necessary.

If that is not a viable solution, I can also fork `ryzen_monitor` for the time being and apply the patch "upstream".

File: app-admin/ryzen_monitor/ryzen_monitor-1.0.5-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+"
+
+LICENSE="AGPL-3 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="app-admin/ryzen_smu"
+
+src_unpack() {
+	unpack "${P}.tar.gz"
```
Comment (Flowdalic) [inline app-admin/ryzen_monitor/ryzen_monitor-1.0.5-r1.ebuild:24]: fwiw: unpack can take multiple arguments

**Comment by thesamesam** [comment]
given
>ryzen_monitor already contains a copy of the ryzen_smu source code. That is just replaced with the newer code from the current, now incompatible ryzen_smu-0.1.5.

I can live with this, but it's not ideal at all :(

**Comment by negril** [comment]
The upstream is `ryzen_monitor` which isn't overly active is seems, because they just pretty_print `ryzen_smu`. It' likely an extended version of the `monitor_cpu` example build the same way https://gitlab.com/leogx9r/ryzen_smu/-/blob/master/userspace/Makefile.

I can also modify the `ryzen_smu` ebuild to create a library and export the header file. If that is preferable. I just need someone to make a decision that I can implement. This being the easiest one and suggested in the upstream bug at https://github.com/hattedsquirrel/ryzen_monitor/issues/25.

**Comment by thesamesam** [comment]
ok, it's janky, but I can live with it if you elaborate on the rationale a bit more in the commit msg too, and maybe we'll reassess if it doesn't get fixed upstream in future too

**Pull request closed without merging**